commit 7eed691cc2b6c5dbb6066ee1251606a744c7f05c
Author: Arjun Shankar <arjun@redhat.com>
Date:   Wed Jul 2 17:11:32 2025 +0200

    stdlib/Makefile: Remove deleted test's libm dependency
    
    tst-qsort5 was deleted in 709fbd3ec3595f2d1076b4fec09a739327459288.
    Therefore remove its redundant libm dependency.
    
    Reviewed-by: Florian Weimer <fweimer@redhat.com>

Conflicts:
	stdlib/Makefile
	  (Fixup context)

diff --git a/stdlib/Makefile b/stdlib/Makefile
index fe663f3bb8ee1e00..81bdce4f6d90d291 100644
--- a/stdlib/Makefile
+++ b/stdlib/Makefile
@@ -635,7 +635,6 @@ $(objpfx)tst-setcontext3.out: tst-setcontext3.sh $(objpfx)tst-setcontext3
 		 $(common-objpfx)stdlib/; \
 	$(evaluate-test)
 
-$(objpfx)tst-qsort5: $(libm)
 $(objpfx)tst-getrandom2: $(shared-thread-library)
 $(objpfx)tst-getenv-signal: $(shared-thread-library)
 $(objpfx)tst-getenv-thread: $(shared-thread-library)
